service-alerts-connector/service-alerts-api.yaml

```diff
@@ -0,0 +1,303 @@
+      description: Get all planned service outages active in the last 7 days with social media fields
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ServiceAlertV0'
+  /v1/coct-service_alerts-all-unplanned.json:
+    get:
+      tags:
+        - "v1"
+      summary: All unplanned outages
+      description: Get all unplanned service outages going back several years with social media fields.
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ServiceAlertV1'
+  /v1/coct-service_alerts-all-planned.json:
+    get:
+      tags:
+        - "v1"
+      summary: All planned outages
+      description: Get all planned service outages going back several years with social media fields.
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ServiceAlertV1'
+  /v1/alerts/{alert_id}.json:
+    get:
+      tags:
+        - "v1"
+      summary: Get a specific service alert by ID
+      description: Retrieve detailed information about a service alert by its unique ID
+      parameters:
+        - name: alert_id
+          in: path
+          description: ID of the service alert to retrieve
+          required: true
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceAlertV1'
+components:
+  schemas:
+    ServiceAlertV0:
+      type: object
+      properties:
+        Id:
+          type: integer
+          description: Unique identifier for service alert
+          example: 23128
+        service_area:
+          type: string
+          description: Type of service affected
+          example: "Water & Sanitation"
+        title:
+          type: string
+          description: Title of service alert
+          example: "Water Off"
+        description:
+          type: string
+          description: More detailed description of nature of outage, and its effects.
+          example: "Leak on Main"
+        area:
+          type: string
+          description: High level description of the area affected. Often the suburb.
+          example: "Sonkring"
+        location:
+          type: string
+          description: More fine-grained description of the area affected. Often the street address.
+          example: "Woudenberg street"
```

Comment:
Based on the coding test for the Data Scientist position you guys have a lot more experience with location data than I do, but it could be very useful to include something like that in V2, rather than human-readable descriptions

Reply:
I should say, in addition to human readable descriptions

Reply:
That's certainly in the works - has required a change in that upstream form. I'll create an issue on GH to track it more visibly - https://github.com/cityofcapetown/service-alerts-connector/issues/2